rllib/tuned_examples/maml/halfcheetah-rand-direc-maml.yaml

```diff
@@ -0,0 +1,25 @@
+halfcheetah-rand-direc-maml:
+    env: ray.rllib.examples.env.halfcheetah_rand_direc.HalfCheetahRandDirecEnv
+    run: MAML
+    stop:
+        training_iteration: 2001    
```

Comment:
If these are actually learning, could you add some decent stopping criteria here? Like episode_reward_mean and training_iteration (or better timesteps_total).

Reply:
Above comment^^